docs/src/main/sphinx/connector/pulsar.rst

```diff
@@ -0,0 +1,270 @@
+=================
+Pulsar connector
+=================
+`Apache Pulsar <https://pulsar.apache.org/en/>`_ is a cloud-native, distributed messaging and streaming platform.
+
+This connector allows the use of Pulsar topics as tables in Trino, such that each Pulsar message
+in a Pulsar topic is presented as a row in Trino. And Pulsar topic schema is mapped to Trino
+table schema.
+
+Topics in Pulsar are stored as segments in `Apache BookKeeper <https://bookkeeper.apache.org/>`_. Each topic segment is replicated to some BookKeeper nodes,
+which enables concurrent reads and high read throughput. You can configure the number of BookKeeper nodes,
+and the default number is 3. In Trino Pulsar connector, data is read directly from BookKeeper,
+so Trino workers can read concurrently from horizontally scalable number BookKeeper nodes.
+
+This connector is **read-only** for now. It can only read data from Pulsar topics,
+but cannot create topic or write to exiting topic yet.
+
+To configure the Pulsar connector, create a catalog properties file ``etc/catalog/pulsar.properties``
+A sample is provided:
+
+.. code-block:: text
+
+    connector.name=pulsar
+    pulsar.broker-service-url=http://localhost:8080
```

Comment:
localhost? It looks like trino discover uri. Maybe let's use some placeholder instead.

Reply:
It looks like this is actually the rest port used by Pulsar. We will need to think of how to deal with this. I think traditionally Trino was an internally managed service that shipped with Pulsar so they likely gave Trino different ports. If we assume a Trino centric point of view, we would likely want to change the port of Pulsar.

https://pulsar.apache.org/docs/en/next/standalone-docker/#use-pulsar-in-docker

Reply:
this is actually the url of Pulsar Admin endpoint, and default value will most likely used in standalone deployment which is used for experimenting/exploring. Will specify in doc that user should change to real url when used in non-test deployment.